extensions/2.0/Khronos/KHR_materials_ior/README.md

```diff
@@ -0,0 +1,144 @@
+# KHR\_materials\_ior
+
+## Contributors
+
+* Tobias Haeussler, Dassault Systemes [@proog128](https://github.com/proog128)
+* Bastian Sdorra, Dassault Systemes [@bsdorra](https://github.com/bsdorra)
+* Ben Houston, ThreeKit [@bhouston](https://github.com/bhouston)
+* Richard Sahlin, IKEA [@rsahlin](https://github.com/rsahlin)
+* Norbert Nopper, UX3D [@UX3DGpuSoftware](https://twitter.com/UX3DGpuSoftware)
+* Ed Mackey, AGI [@emackey](https://twitter.com/emackey)
+
+Copyright (C) 2018-2020 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Experimental
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+The dielectric BRDF of the metallic-roughness material in glTF uses a fixed value of 1.5 for the index of refraction. This is a good fit for many plastics and glass, but not for other materials like water or asphalt, sapphire or diamond. This extensions allows users to set the index of refraction to a certain value.
+
+## Extending Materials
+
+The index of refraction of a material is configured by adding the `KHR_materials_ior` extension to any glTF material. 
+
+```json
+{
+    "materials": [
+        {
+            "extensions": {
+                "KHR_materials_ior": {
+                    "ior": 1.4,
+                }
+            }
+        }
+    ]
+}
+```
+
+| |Type|Description|Required|
+|-|----|-----------|--------|
+| **ior** | `number` | The index of refraction. | No, default: `1.5`|
+
+Typical values for the index of refraction range from 1 to 2. In rare cases values greater than 2 are possible. The following table shows some materials and their index of refraction:
```

Comment:
For the specification, I think we should be more explicit on the allowed range for ior.  If a value of -1 is specified in this extension, most implementations will result in nan values all throughout the material.  If this is a defined constraint, we could have the validator at least flag this value as illegal.

Minimum value: 1.0 ?
Maximum value: 3.0 ?

Reply:
The schema defines a minimum of 1.0 (although the math would work for any value >0). There is no maximum, because it will make conversions from other material models (e.g. KHR_materials_pbrSpecularGlossiness) more complicated.

Reply:
Do we want to allow materials less dense than air?  Also, for an underwater scene, the air bubbles in the scene would be less than 1.0 IOR due to light exiting the water, right?  Could the set the limit as >0 instead of >= 1.0?

Reply:
I think I'll withdraw this comment.  After a bit more research, it looks like a pure vaccuum is 1.0, and air is actually 1.0002.  So presumably values less than 1.0 are not physically possible.